tests/unit/test_aml_wide_deep.py

```diff
@@ -0,0 +1,69 @@
+from reco_utils.aml.wide_deep import parse_args
```

Comment:
I haven't seen many times peoples testing argparse, it is a mature library. If you really want to test that function, maybe you could just test the exceptions.

Reply:
Removing from the next commit.